_api-reference/cluster-api/cluster-state.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: default
+title: Cluster state
+nav_order: 55
+parent: Cluster APIs
+---
+
+# Cluster state API
+
+The `/_cluster/state` API retrieves the current state of the cluster, including metadata, routing table, nodes, and other components. This API is primarily used for monitoring, debugging, and internal purposes. It gives a snapshot of various parts of the cluster's state as maintained by the cluster manager node.
+
+## Endpoints
+
+```json
+GET /_cluster/state
+GET /_cluster/state/{metric}
+GET /_cluster/state/{metric}/{target}
+```
+
+## Path parameters
+
+| Parameter   | Data type         | Description |
+| ----------- | ----------------- | ----------- |
+| `metric`    | String or list    | Optional. Comma-separated list of metrics to include in the response. See [metric options](#metric-options) for list of available options. Default is `_all` |
+| `target`     | String or list    | Optional. Comma-separated list of index names, data streams, and index aliases to limit the scope of the response. |
+
+## Metric options
+
+You can limit the information returned by the API using the `metric` path parameter. The following options are available:
+
+- `_all`: Returns all metrics. _(Default)_
+- `blocks`: Includes information about index-level and global blocks.
+- `metadata`: Returns cluster metadata, including settings, index mappings, and templates.
+- `nodes`: Returns information about the nodes in the cluster.
+- `routing_table`: Provides the routing information for all shards.
+- `master_node`: Displays the ID of the currently elected cluster manager node.
+- `version`: Displays the current cluster state version.
+
+## Query parameters
+
+The following table lists the available query parameters. All query parameters are optional.
+
+| Parameter        | Data type | Description |
+| ---------------- | --------- | ----------- |
+| `local`          | Boolean   | If `true`, retrieves the state from the local node instead of the cluster manager node. Default is `false`. |
+| `master_timeout` | Time      | The timeout duration for connecting to the cluster manager node. Default is `30s`. |
+| `flat_settings`  | Boolean   | If `true`, returns settings in a flat format. Default is `false`. |
+| `wait_for_metadata_version` | Integer | Waits until the metadata version is equal or greater than this value before responding. |
+| `wait_for_timeout` | Time   | Specifies how long to wait when using `wait_for_metadata_version`. Default is `30s`. |
+| `ignore_unavailable` | Boolean | Whether to ignore missing or closed indexes. Default is `false`. |
+| `expand_wildcards` | String | Whether to expand wildcard expressions for index names. Options: `open` `closed`, `hidden`, `none`, `all`. Default is `open`. |
```

Comment:
Can you take expand_wildcards description from another API page? It should have descriptions for all these values.